intranetunits/models.py

```diff
@@ -38,13 +38,6 @@ class IntranetUnitsReportsIndexPage(BasePage):
     content_panels = Page.content_panels + BasePage.content_panels
     subpage_types = ['intranetunits.IntranetUnitsReportsPage']
 
-    def clean(self):
-        """
-        Make sure page titles adhere to strict
-        formatting policy.
-        """
-        enforce_name_as_reports(self.title)
-
```

Comment:
Since both `enforce_name_as_reports` and `enforce_name_as_year` are now unused in this file, we should get rid of them from the import at the top of the page (line 13).

Reply:
Looks like `enforce_name_as_reports` is being used by `GroupMemberRole` still. I am unsure what that class is using the reports function for.